### PR TITLE
Initial/basic website

### PR DESCRIPTION
**Pull request by rebbrunner**
Add current website to repo -
.gitignore was based off linked article as well as several posts in stackexchange.
Database and env file are in ignore, will have to be created on each unit.
Decoupled SECRET_KEY, DEBUG, ALLOWED_HOSTS, and DATABASES.  No testing done on if this setup works on server yet.
No commits were made until I decoupled the above settings from settings and the database was ignored.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,23 +1,139 @@
...
+ # Package Files #
+*.jar
+*.war
+*.nar
+*.ear
+*.zip
+*.tar.gz
+*.rar
+ # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
+hs_err_pid*
```
Comment (psybers) [inline .gitignore]: all the above ignores are for Java projects - we can remove these

**Review by psybers — CHANGES_REQUESTED** [review]
We arent ignoring media/ ?

**Comment by psybers** [comment]
static/website/people was removed on the server - dont commit these

**Comment by psybers** [comment]
remove default parent/child models and all associated data for them

**Comment by psybers** [comment]
Your commits are still not linking properly to your GH user.

**Comment by rebbrunner** [comment]
I removed the given lines from .gitignore and added media/.
I removed the people subdirectory from static
I removed the default parent/child and ran manage.py makemigrations and migrate.
I don't know what the problem is with my GH account I checked my local and global commit and my name and email are displaying. :S

**Comment by psybers** [comment]
The media/ files are still in the commit.

**Comment by psybers** [comment]
Since we are pushing the 'first iteration' of this website, there probably doesnt need to be a migrations folder/files.

**Comment by rebbrunner** [comment]
I unstaged all changes and re-added them then committed.
Updated user.name to Rebecca Brunner and email to my school email

**Comment by rebbrunner** [comment]
Trying to figure out why my account is not linking correctly.

EDIT:

Alright, I believe my account is now linking - I had the wrong email under github.

**Comment by rebbrunner** [comment]
Removed migrations folder for initial commit

**Comment by rebbrunner** [comment]
Also, on a side note, I have fixed the people pages bug, added the given groups (could not determine how to automatically generate groups at the moment), added sorting by groups, template tags for the purpose of sorting via groups, and a context processor to store constants to use in templates.  All these changes are stored in my other repository and I will migrate them over to the main repo/server once we have the initial backup on GitHub setup.

**Comment by psybers** [comment]
I'm still unsure if we should be pushing the static files. Read up a bit.

https://docs.djangoproject.com/en/2.1/howto/static-files/
https://docs.djangoproject.com/en/2.1/howto/static-files/deployment/
https://docs.djangoproject.com/en/2.1/ref/contrib/staticfiles/#django-admin-collectstatic

**Comment by rebbrunner** [comment]
Ah okay, so typically in production static files can be stored in another location and are transferred over to the server's static folder via command/service.  I will add the static files to the .gitignore.  Also, I apologize for how long this is taking.  I am completely new to deploying apps in a production environment and am learning as we go!

**Comment by rebbrunner** [comment]
Static files added to .gitignore.

**Comment by psybers** [comment]
We have some custom static files though - how do we handle those?  Probably we dont want to just ignore them.

**Review by psybers — APPROVED** [review]

**Pull request merged**